examples/python_rust_compiled_function/src/ffi.rs

```diff
@@ -71,7 +71,7 @@ pub fn rust_series_to_py_series(series: &Series) -> PyResult<PyObject> {
     let array = series.to_arrow(0);
 
     // acquire the gil
-    let gil = Python::acquire_gil();
+    let gil = Python::with_gil();
```

Comment:
This does not look correct. Is this example built in your CI? [`with_gil`](https://docs.rs/pyo3/latest/pyo3/marker/struct.Python.html#method.with_gil) takes a closure which gets the GIL token passed as an argument.

Reply:
I made this change because I got a deprecation warning locally.

Reply:
The deprecation is intentional, but `Python::with_gil` just works differently. This PR already contains other places where that change is made correctly, so you should be able to copy from there.

Reply:
@jjerphan this should be:

```
pub fn rust_series_to_py_series(series: &Series) -> PyResult<PyObject> {
    // ensure we have a single chunk
    let series = series.rechunk();
    let array = series.to_arrow(0);

    Python::with_gil(|py| {
        // import pyarrow
        let pyarrow = py.import("pyarrow")?;

        // pyarrow array
        let pyarrow_array = to_py_array(py, pyarrow, array)?;

        // import polars
        let polars = py.import("polars")?;
        let out = polars.call_method1("from_arrow", (pyarrow_array,))?;
        Ok(out.to_object(py))
    })
}
```